joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -881,20 +929,61 @@ class AstCreator(filename: String, global: Global)
       })
       .toSeq
 
-    Ast(seqNodes)
+    val seqMethodPar = localVarList
+      .map(localVar => {
+        val varSymbol = localVar.getSymbol()
+        NewMethodParameterIn()
+          .name(varSymbol.getText)
+          .code(varSymbol.getText)
+          .lineNumber(varSymbol.getLine)
+          .columnNumber(varSymbol.getCharPositionInLine)
+      })
+      .toSeq
+
+    val blockNode = NewBlock().typeFullName(Defines.Any)
+    val ast       = Ast(seqMethodPar).withChild(Ast(seqNodes))
+    blockAst(blockNode, List[Ast](ast))
   }
 
   def astForBodyStatementContext(ctx: BodyStatementContext): Ast = {
     astForStatementsContext(ctx.compoundStatement().statements())
+    // TODO rescue else and ensure to be implemented
   }
 
   def astForMethodDefinitionContext(ctx: MethodDefinitionContext): Ast = {
-    val astMethodName  = astForMethodNamePartContext(ctx.methodNamePart())
+    val astMethodName = astForMethodNamePartContext(ctx.methodNamePart())
+    val callNode      = astMethodName.nodes.filter(node => node.isInstanceOf[NewCall]).head.asInstanceOf[NewCall]
+    // there can be only one call node
+
+    val classPath = classStack.toList.mkString(".") + "."
+    val methodNode = NewMethod()
+      .code(callNode.code)
+      .name(callNode.name)
+      .fullName(classPath + callNode.name)
+      .order(1)
```

Comment:
Remove manual setting of the order field, this should be taken care of automatically.

Reply:
Done